src/parsers/chrono/chrono.js

```diff
@@ -6,8 +6,8 @@ import chrono from 'chrono-node';
  */
 const dayPeriodsParser = new chrono.Parser();
 dayPeriodsParser.pattern =
-  () => new RegExp('midnight|morning|in the morning|noon|' +
-    'afternoon|in the afternoon|evening|in the evening|night|at night', 'i');
+  () => new RegExp('\\b(?:midnight|morning|in the morning|noon|afternoon|' +
+    'in the afternoon|evening|in the evening|night|at night)\\b', 'i');
```

Comment:
The `\b` means "edge of a word". I think it's always a good practice to have this in regexps matching words. For example we don't want to match `nightly` (I haven't checked we did before the change though, tell me if you'd like me to check this.)